### ISSUE TITLE
Fine tune only landmark

### ISSUE THREAD

**Issue by pribadihcr** [CongWeilin/mtcnn-caffe#52]
Hi @CongWeilin,

I want to fine tune the 48net only for landmark. I got the Nan loss
I0815 12:38:38.928587  6445 solver.cpp:219] Iteration 0 (9.95635e+13 iter/s, 0.0408612s/500 iters), loss = 2925.91
I0815 12:38:38.928714  6445 solver.cpp:238]     Train net output #0: ClassifyLoss = 0.693147 (* 1 = 0.693147 loss)
I0815 12:38:38.928751  6445 solver.cpp:238]     Train net output #1: LandmarkLoss = 2925.22 (* 1 = 2925.22 loss)
I0815 12:38:38.928787  6445 solver.cpp:238]     Train net output #2: RegressionLoss = 0 (* 0.5 = 0 loss)
I0815 12:38:38.928815  6445 solver.cpp:238]     Train net output #3: cls_Acc = 0
I0815 12:38:38.928854  6445 sgd_solver.cpp:105] Iteration 0, lr = 0.0001
I0815 12:38:46.481483  6445 solver.cpp:219] Iteration 500 (66.2002 iter/s, 7.55285s/500 iters), loss = nan
I0815 12:38:46.481575  6445 solver.cpp:238]     Train net output #0: ClassifyLoss = 0.693147 (* 1 = 0.693147 loss)
I0815 12:38:46.481603  6445 solver.cpp:238]     Train net output #1: LandmarkLoss = nan (* 1 = nan loss)
I0815 12:38:46.481631  6445 solver.cpp:238]     Train net output #2: RegressionLoss = 0 (* 0.5 = 0 loss)
I0815 12:38:46.481654  6445 solver.cpp:238]     Train net output #3: cls_Acc = 0
I0815 12:38:46.481678  6445 sgd_solver.cpp:105] Iteration 500, lr = 0.0001
I0815 12:38:53.950376  6445 solver.cpp:219] Iteration 1000 (66.9445 iter/s, 7.46888s/500 iters), loss = nan
I0815 12:38:53.950469  6445 solver.cpp:238]     Train net output #0: ClassifyLoss = 0.693147 (* 1 = 0.693147 loss)
I0815 12:38:53.950506  6445 solver.cpp:238]     Train net output #1: LandmarkLoss = nan (* 1 = nan loss)
I0815 12:38:53.950537  6445 solver.cpp:238]     Train net output #2: RegressionLoss = 0 (* 0.5 = 0 loss)
I0815 12:38:53.950565  6445 solver.cpp:238]     Train net output #3: cls_Acc = 0
I0815 12:38:53.950589  6445 sgd_solver.cpp:105] Iteration 1000, lr = 0.0001
